build/roslib.js

```diff
@@ -574,7 +574,7 @@ Param.prototype.get = function(callback) {
  *
  * @param value - value to set param to.
  */
-Param.prototype.set = function(value) {
+Param.prototype.set = function(value, callback) {
```

Comment:
Here and in other places it'd be nice to keep the behavior if the callback is not provided via `callback = callback || function() {};`